pages/cn/services/Spark-2-3-1-2-2-1-2/job-scheduling/index.md

```diff
@@ -83,7 +83,7 @@ $ cat options.json
         "role": "dispatcher"
     }
 }
-$ dcos package install spark --options=options.json
+dcos package install spark --options=options.json
```

Comment:
So not sure what we want to do about cases like this. Here, the `$` distinguishes a command that should be entered and everything else is the output of that command.

I think for these cases, the command and output should just be split up between multiple code blocks so the copy to clipboard functionality works correctly (to both remove `$` and not copy the output to the clipboard). Not sure when we want to do this, but we would need to look at every code block in our docs 😬

Reply:
*head desk*